### ISSUE TITLE
Directives not working after ssr:static

### ISSUE THREAD

**Issue by demyan1411** [universal-vue/uvue#33]
In `server.config.js` I add
```
renderer: {
      directives: {
        tooltip: function (vnode, meta) {
          console.log(vnode);
          vnode.text = meta.value;
        }
      },
    }
```
and after yarn ssr:static directive not working

**Comment by chymz** [comment]
Thanks for report !

**Comment by chymz** [comment]
Fixed in `0.1.0-alpha.19`
